apidoc/Titanium/UI/UI.yml

```diff
@@ -1521,8 +1542,108 @@ properties:
         `KEYBOARD_DEFAULT`.
     type: Number
     permission: read-only
+    deprecated:
+        since: "5.2.0"
+        notes: Use KEYBOARD_TYPE_PHONE_PAD instead.
 
-  - name: KEYBOARD_WEBSEARCH
+  - name: KEYBOARD_URL
+    summary: Use a keyboard optimized for entering URLs, with the standard keyboard layout.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType> 
+        properties.
+        
+        On Mobile Web, if you enable `passwordType`, the `keyboardType` value will be overridden with
+        `KEYBOARD_DEFAULT`.
+    type: Number
+    permission: read-only
+    deprecated:
+        since: "5.2.0"
+        notes: Use KEYBOARD_TYPE_URL instead.
+
+  - name: KEYBOARD_TYPE_DECIMAL_PAD
+    summary: Use a keyboard with decimal numbers only, with the pad keyboard layout.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties.
+
+        On iPad, this behaves the same as <Titanium.UI.KEYBOARD_TYPE_DEFAULT>.
+        type: Number
+    permission: read-only
+    platforms: [android, iphone, ipad]
+
+  - name: KEYBOARD_TYPE_ASCII
+    summary: Use an ASCII keyboard, with the standard keyboard layout.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties.
+    type: Number
+    permission: read-only
+    platforms: [android, iphone, ipad]
+
+  - name: KEYBOARD_TYPE_DEFAULT
+    summary: Use the default keyboard, depending on the platform.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties.
+    type: Number
+    permission: read-only
+
+  - name: KEYBOARD_TYPE_EMAIL
+    summary: Use a keyboard suitable for composing email, with the standard keyboard layout.
+    description: |
+        Use with the <Titanium.UI.TextField.keyboardType> and <Titanium.UI.TextArea.keyboardType>
+        properties.
+
+        Note: on Mobile Web, if you enable `passwordType`, the `keyboardType` value will be
+        overridden with `KEYBOARD_TYPE_DEFAULT`.
```

Comment:
Mobileweb stays `KEYBOARD_TYPE_DEFAULT` for now. @cheekiatng or where should he change those?